pkg/admission/permissionclaims/mutating_permission_claims.go

```diff
@@ -89,7 +89,7 @@ func (m *mutatingPermissionClaims) Admit(ctx context.Context, a admission.Attrib
 		return err
 	}
 
```

Comment:
@s-urbaniak @ncdc Regarding #2818, there's been discussion of checking APIexport permissions at admission time.

Is this file the correct place to do so? Can we implement part of the admission in a PR separate from this one to reduce the size?

Reply:
We will need to enable admission control in the view (virtual workspace) framework, then we can check name/namespace restrictions using something like this.